### ISSUE TITLE
Consider making --always-install the default

### ISSUE THREAD

**Issue by devtooligan** [trailofbits/solc-select#197]
I was poking around the code and discovered this great --always-install.  It doesn't show up on the top level of help (and I previously didn't realize it shows up with no description if you do `solc-select use --help`. 

But personally I think solc-select should --always-install by default and we change this option to --never-install

Either way, I think it would be nice to have this option listed on the top help screen.

**Comment by devtooligan** [comment]
This issue got me again today.  

I was thinking if there was a concern about automatically installing it we could accept user input like "Version 0.8.19 is not installed.  Install y/N?"

**Comment by 0xalpharush** [comment]
I agree. We could add an `--offline` flag if users want to opt out of downloading

**Comment by legit-programmer** [comment]
Hi, I up am for solving this issue, please explain what is to be done in brief, as the details given above are not comprehensive...

**Comment by devtooligan** [comment]
It would involve making the --always-install the default.  removing that particular clarg.  and then adding a new clarg called "--offline" which would make it not install by default.
